### DISCUSSION TITLE
Merge the feature branches into the development branch

### DISCUSSION THREAD

**Discussion by maryamabasova** [LianqiangGuan/Tafe230211#10] [General]
Hello Team,

What are the steps for merging the feature branches into the development branch?

**Comment by LianqiangGuan** [comment]
Hi,
 Before merging you could use a revision graph to check you are already on the development branch (the red color shows the current revision). Then right-click TortoiseGit >Merge > choose your feature branch. You can enter your own merge message.
 After merging you could use revision graph again to check the merge is done.
